mempool/src/tests/shared_mempool_test.rs

```diff
@@ -58,97 +69,103 @@ struct SharedMempoolNetwork {
     network_conn_event_notifs_txs: HashMap<PeerId, conn_notifs_channel::Sender>,
     runtimes: HashMap<PeerId, Runtime>,
     subscribers: HashMap<PeerId, UnboundedReceiver<SharedMempoolNotification>>,
-    peer_ids: HashMap<PeerId, PeerId>,
+    /// A mapping of secondary `PeerId` on other network interfaces to the main `PeerId`
+    main_peer_ids: HashMap<PeerId, PeerId>,
```

Comment:
nit: Perhaps replacing the word `main` with `primary` throughout the code might be more familiar to the reader? e.g., primary peer id and secondary peer id? This would also fit with the comment 😄